### ISSUE TITLE
Q: How do you use hooks inside of a component nested in the <Stage>?

### ISSUE THREAD

**Issue by RossKinsella** [michalochman/react-pixi-fiber#142]
### Description
I'm interested in having a `useEffect` hook in a component which renders a `Container`, nested in a `Stage`

eg:

#### Tree
```
<Stage>
  <MyComponent />
</Stage>
```

#### MyComponent.tsx
```
import React, { useEffect } from "react";
import { Container } from "react-pixi-fiber";

const MyComponent = () => {
  useEffect(() => {
     // This line is not being executed
  }, []);

  return (
     <Container />
  );
};
```
### Steps to reproduce

1. Set up a project with a layout similar to the one in my description
2. When you run the code, it will throw this exception:
```
Uncaught TypeError: callback is not a function
    at flushFirstCallback (scheduler.development.js:99)
    at flushWork (scheduler.development.js:216)
    at MessagePort.push../node_modules/react-pixi-fiber/node_modules/scheduler/cjs/scheduler.development.js.channel.port1.onmessage (scheduler.development.js:586)
```

### Additional info

- react-pixi-fiber version: 0.9.3
- React version: 16.9.0
- ReactDOM version: 16.9.0
- PixiJS version: 5.1.2

PS:
I investigated the issue a little and tried to fix it by ensuring the same version of scheduler and react-dom were being used by everything in my project in this fork: 
https://github.com/RossKinsella/react-pixi-fiber/tree/fix/scheduler-dependency
- However, I'm not sure if this is the right direction to take as I didn't find a solution

**Comment by RossKinsella** [comment]
Incase it's relevant, here is the output of `yarn why scheduler`

```
yarn why v1.13.0
[1/4] 🤔  Why do we have the module "scheduler"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "scheduler@0.14.0"
info Reasons this module exists
   - "react-pixi-fiber" depends on it
   - Hoisted from "react-pixi-fiber#scheduler"
info Disk size without dependencies: "160KB"
info Disk size with unique dependencies: "208KB"
info Disk size with transitive dependencies: "236KB"
info Number of shared dependencies: 3
=> Found "react-dom#scheduler@0.15.0"
info This module exists because "react-dom" depends on it.
info Disk size without dependencies: "212KB"
info Disk size with unique dependencies: "260KB"
info Disk size with transitive dependencies: "288KB"
info Number of shared dependencies: 3
=> Found "react-reconciler#scheduler@0.13.6"
info This module exists because "react-pixi-fiber#react-reconciler" depends on it.
info Disk size without dependencies: "128KB"
info Disk size with unique dependencies: "176KB"
info Disk size with transitive dependencies: "204KB"
info Number of shared dependencies: 3
✨  Done in 0.73s.
```

**Comment by michalochman** [comment]
Hi @RossKinsella thanks for reporting. We are finalising hooks support in #127, where we have actually downgraded scheduler to `0.13.6`

**Comment by michalochman** [comment]
Fixed in #127
